S4TF_Tutorial_3.ipynb

```diff
@@ -0,0 +1,1769 @@
X6ekkoL6/BNxwLAwIEDy/bBFYVbVa4SaiC4mujEzdTgvvbBoTdpU88c9sCZM9KO\nNZ9znkpLmybidVxNc3hEnz5rUj66umPdpH0mnybdY+c+KV+OGjmmuviU//u/8mky3Sf1+TWHvvTs\n2dPbjs+NqxU+Znz8Q8M1fPpy27Ee3Cfeeb00H1XNUIxm1ko50qa1Z8xYU+iLswsAyesSh0FlqWLH\n1fcA4Omnn7Y2h3D5qlC2B3/GtPPH7V5//fVM+2Jivv689957ePzxxwGs/VmPPvpoa/N3jsNtXDjL\nkVtBmEMd2D+fffbZiXa+0AvmiCOSdTWmT59ubc6oUW1WrlyZeB2q85CsF03zi7IxZqEx5rmi/S4K\nAdJ9UKh2cmOx2Y0AjqxVJ5XqUor9cf8qQXXSetRCJ4BqpRVRrSihlNNKpahOWo9a+ZQsdChGWUQG\nAhiKQn3snsaY0m3UIgA9PeucAeAMIPnQmtJYanmnpjppHWp9R69aaR1i04rqJF5ivv64M0lK42ia\nX5RLiMgmAO4EcI4xZhUvM4Xf0Mv+jm6MGWWMaTPGtG2xxRYVdVapDrWM/VGdtA61jhFTrbQOMWpF\ndRIntYhRpm1X7FO4AJTSOJotRhkisi4K4rvZGHNX8e3FItLbGLNQRHoDWOLfQsfgeCg3TozjATme\n08UXE5p2oDlmxme720irzMf75ThpN+0XV+ZLi1GuZmW+Wgiu3jpJg2O/0lLeuDFjSpJaOaZ6aOWf\n//xn2ffdZxY4dpi/m9dcc02i3be//W1r8/d0k002SbTjY8Zxzfx+WvpB37aAZFwq226cIKfGe+21\n16wdOghwU56F/MoWu1ZK/jM0FVtaerhK0l51hNNPPz3xevbs2da+//77K95+6LM4fG5nzZpV8X5j\nvv589NFHmDdvHgDgu9/9bmLZz372M2vz996NJedlnGrOjWfn9dLGEueff761/+M//sPaP/rRmnDr\nRx99NLHOgQceaG2uEltt3PhsN6Wvj5DvYdP8oiyFT3M9gJnGmN/TorEATinapwCoXbS4UlVqEfuj\nOmk9ahUjplppPVQrSii1iFFWnbQezRajvDeAkwC8JCIvFN+7AMDFAG4XkdMAvAbg2Np0UakFNbhT\nU520IDW6o1ettCCqFSUUvf4oIcTyi3K7A2VjzJMAfL+RH1Dd7qyN+/M8T0mkpbfiqU1flSq3HW87\nNESD++fe7fimstxp0h133LFsH9zpl5hDLxqtE6U21MJR1UsrnO6PpwM5ZRfg/4xHHXVU4vX3vvc9\na48ZM8babijVsmXLrN27d++y/XFh38E+xa0QxukMeZ3dd9890Y7TSz322GNlt+1ugxk7dmzitRsC\nUI7YtRIachHSnn3xoYcemljG0+kjR4609gknnBC034suusjabvjQOeecY+0hQ4YEba8a8HVy+fLl\nFW8v5utPt27dcOqppwIARo0alVjGaQH5OLifp1evXtbm7yxXdASA7t27W5tDqdxr/W9/+9uyNsfe\nuyGG//M//4NyuOOKSs+F+5lCw7tC9ts0A2Wl9ShNaShKGqoTJRTVihKKakUJISad6EA5p8Ryp6bE\njepECUW1ooSiWlFCiEUn0Q+U0yrzDRgwwLseP8nO0xPuE5m+OxZ+Mt6dqnBfl+sbkJxq5WkVnooB\n/Nk73L65Ff2yUkq7oihpNLtO2HdweETW9E8XX3xxWTsN/t5zH9IyKrDtVhFMq0YagrtfDkXjSmP3\n3Xdfol17oRexa+Xdd9/FhAkTAKx9TPma0LVrV2u7lSz5msLHyq2KOHfuXGtfdtll1uYsBADQo0cP\naz/wwAPW/sMf/mDt/fbbL7FOqO6ykBZqwtc8Pg5Z9xOzVhiumgsAEydOtDbn5uZqwkAyawwfOzdD\nDo8R0o4/Z2fyHX8O9wD8oTkdDUEqwX3lMA83lNSXIYd9IbD298YlJp3E0Qul7mTJTygiw0XkZRGZ\nWywH6i4/V0RmiMhUEXlYRPx3MkpTkDWPpWolf6hPUUJRn6KEEItP0YFyDsmSdkVEOgG4GsAIADsA\nOF5EdnCaPQ+gzRizM4A7AFxag+4rdSJreh7VSv5Qn6KEkiU9nGolf8TkU3SgnFMy3KkNAzDXGDPP\nGPMxgFsBHMENjDGPGmNKKQUmAuhb9Y4rdSXjL8qqlRyiPkUJRX2KEkIsPiXKGOW0qnhMWgUYX3yw\nm27pnXfesTbHJYdWLGLc2GWOJ3z//fet7Vay4Vgd7rcbk+zGQWUlJfanu4hMptejjDGl/Dh9AHBZ\noQUAkjmpkpwG4B8VdVRpKBl1AkSileuvv97ad911l7X5uwiEp4XMQlosa73gOMu33347sYzjtdlP\n7r333h3aR+w+5eOPP8b8+fMBwP4vsWTJmmJtHEfuXis4TpSvD/369Uu0O/HEE6298847W/uhhx5K\ntHv66aet/dJLL1l7n332sTbHOAPJ+Gq+VlQaN9weHJN6yCGHVLQtj1ai9Ck//vGPE69vueUWa3OV\nPXeMwGMTHge454njhfl676ax5e2zv2K9cspKl2r4ON84yI099sUo+57t8hGTT4lyoKzUHs8UxlJj\nTFul2xaREwG0Adi3vbZK3NRSJ4BqpZVQn6KEUkYr6lOUtYjFp+hAOYdkfJr0TQD800nf4nvutg8E\n8BMA+xpj/BUWlOip4Klj1UrOUJ+ihKJaUUKISSdRDpT/P3vvHn9VXeX/PxekiTcUUERQwcQLmoLi\n/UapDTqTOKNjYqVOllONTpbjN5pGp6yZ6usvmy5eQjMtS0dTv1KRaCZiXgFFFAhEEEVRRBTDO/r+\n/XH22az95uzzeX/O9X3OXs/H4/P4rHP2e+/9Pme/ztrvvd9rr6WrAPmpfEJDIk466aTUfu2111Jb\np4rz95UXKK7b+PvVtn9QdShH//79U3vs2PyLIT3d5/fH70c91
```

Comment:
Remove parentheses around `if`/`guard`/`for`/`while` conditions. Same elsewhere.
```suggestion
        "    if correctLabelIndex != predictedLabelIndex\n",
```